.travis.yml

```diff
@@ -8,6 +8,11 @@ addons:
       - jq
 before_install:
   - npm install
+  # Install PhantomJS 2.1.1 manually
+  - "export PATH=$PWD/travis_phantomjs/phantomjs-2.1.1-linux-x86_64/bin:$PATH"
+  - "if [ $(phantomjs --version) != '2.1.1' ]; then rm -rf $PWD/travis_phantomjs; mkdir -p $PWD/travis_phantomjs; fi"
+  - "if [ $(phantomjs --version) != '2.1.1' ]; then wget https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2 -O $PWD/travis_phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2; fi"
+  - "if [ $(phantomjs --version) != '2.1.1' ]; then tar -xvf $PWD/travis_phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2 -C $PWD/travis_phantomjs; fi"
```

Comment:
Presumably this also takes some time to download and could flake out (i've seen bitbucket downloads flake in the past).  Combine this with the ugliness and I'd like to make sure we only land this if we are somewhat confident that it will fix out flaky test.  If we have come confidence than lgtm.